cmd/root.go

```diff
@@ -117,7 +118,7 @@ func NewRootCmd() *RootCmd {
 	flags.StringVar(&rootOpts.KernelConfigData, "kernelconfigdata", rootOpts.KernelConfigData, "base64 encoded kernel config data: in some systems it can be found under the /boot directory, in other it is gzip compressed under /proc")
 	flags.StringVar(&rootOpts.ModuleDeviceName, "moduledevicename", rootOpts.ModuleDeviceName, "kernel module device name (the default is falco, so the device will be under /dev/falco*)")
 	flags.StringVar(&rootOpts.ModuleDriverName, "moduledrivername", rootOpts.ModuleDriverName, "kernel module driver name, i.e. the name you see when you check installed modules via lsmod")
-	flags.StringVar(&rootOpts.BuilderImage, "builderimage", rootOpts.BuilderImage, "docker image to be used to build the kernel module and eBPF probe. If not provided, the default image will be used.")
+	flags.StringVar(&rootOpts.BuilderImage, "builderimage", driverbuilder.BuilderBaseImage, "docker image to be used to build the kernel module and eBPF probe. If not provided, the default image will be used.")
```

Comment:
If this changes the `BuilderBaseImage` directly the previous change that creates a `CustomBaseImage` clashes a bit with that. Which do you think it's better so we can keep only one?

Reply:
We saw together, I'm fixing that too.